packages/exampleReactNative/__tests__/App-test.tsx

```diff
@@ -0,0 +1,14 @@
+/**
```

Comment:
Brilliant. Thanks for adding the tests as well - sadly for the UI parts we are currently not doing a good enough job, having these certainly helps.